### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by JihemNC** [ivan-fr/GoDofus#1]
C'est dofus 2.0  ?

**Comment by ivan-fr** [comment]
oui, dofus 2.60, cependant, il gère uniquement la connexion au serveur d'authentification. c'est un bot qui est destiner à faire de la forgemagie uniquement.
